recipes/dask-deltatable/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "dask-deltatable" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dask-deltatable-{{ version }}.tar.gz
+  sha256: 25885d186cf05525c21cd5895f599a0b68427ff3f88878eb926dc4a5c6c5e522
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+  run:
+    - python >=3.9
+    - dask-core
+    - deltalake
+    - fsspec
+    - pyarrow
+
+test:
+  imports:
+    - dask_deltatable
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  summary: Dask + Delta Table
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - charlesbluca
```

```suggestion
    - charlesbluca
    - jacobtomlinson
```

Can you add me here? Might also want to add @jrbourbeau and @fjetter.

---

Thanks @charlesbluca @jacobtomlinson. I'm happy to be added